### PR TITLE
Adds taxprofiler databases

### PR DESCRIPTION
**Pull request by jfy133**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,13 +46,42 @@ Data was downloaded with nf-core/fetchNGS 1.5 (with Nextflow 21.10.06):
...
+```
+
+#### MALT
+
+```bash
+wget https://software-ab.informatik.uni-tuebingen.de/download/megan6/megan-nucl-Feb2022.db.zip
+unzip megan-nucl-Feb2022.db
+malt-build -i *.gz -s DNA -d taxprofiler-testdb -t 8 -st 4 -a2t megan-nucl-Feb2022.db
+```
+
```
Comment (jfy133) [inline README.md:84]:
```suggestion
#### Kraken2

```bash
kraken2-build --download-taxonomy --db taxprofiler-testdb
gunzip ../raw/*.gz
kraken2-build --add-to-library ../raw/GCF_000146045.2_R64_genomic.fna --db taxprofiler-testdb/
kraken2-build --add-to-library ../raw/NC_012920.1.fa --db taxprofiler-testdb/
kraken2-build --build --db taxprofiler-testdb/
kraken2-build --clean --db taxprofiler-testdb/
```
```

**Review by sofstam — APPROVED** [review]
Looks good to me!

File: README.md [hunk 2]
```diff
@@ -43,13 +46,50 @@ Data was downloaded with nf-core/fetchNGS 1.5 (with Nextflow 21.10.06):
...
+### fasta
 
 One of the files was converted to FASTA file with seqtk 1.3-r106
 
 ```bash
 seqtk seq -a  ERX5474930_ERR5766174_1.fastq.gz > ERX5474930_ERR5766174_1.fa.gz 
 ```
 
+FASTQ files are stored under `data/fasta/`
```
Comment (Midnighter) [inline README.md:59]:
```suggestion
FASTA files are stored under `data/fasta/`
```

File: README.md [hunk 3]
```diff
@@ -43,13 +46,50 @@ Data was downloaded with nf-core/fetchNGS 1.5 (with Nextflow 21.10.06):
...
+```
+#### Kraken2
+
+```bash
+kraken2-build --download-taxonomy --db taxprofiler-testdb
+gunzip ../raw/*.gz
+kraken2-build --add-to-library ../raw/GCF_000146045.2_R64_genomic.fna --db taxprofiler-testdb/
+kraken2-build --add-to-library ../raw/NC_012920.1.fa --db taxprofiler-testdb/
+kraken2-build --build --db taxprofiler-testdb/
+kraken2-build --clean --db taxprofiler-testdb/
```
Comment (Midnighter) [inline README.md:92]:
````suggestion
kraken2-build --clean --db taxprofiler-testdb/
```
````

**Review by Midnighter — CHANGES_REQUESTED** [review]
Thank you for handling the test data 🙂 

I suggest two small improvements to the readme.

I was also wondering when adding test data for the Bracken module, is there really a point in storing the directories and files in addition to the compressed tar archives? Since you provide the commands to replicate, it seems redundant.

**Comment by jfy133** [comment]
> I was also wondering when adding test data for the Bracken module, is there really a point in storing the directories and files in addition to the compressed tar archives? Since you provide the commands to replicate, it seems redundant.

Whoops, correct!

**Review by Midnighter — APPROVED** [review]

**Pull request merged**